_posts/2016-10-04-5-useful-things-you-wouldn't-think-Siri-can-do-for-you.markdown

```diff
@@ -0,0 +1,80 @@
+---
+
+# 5 useful things you wouldn't think Siri can do for you
+
+__Siri is your personal assistant in your pocket. Personally, I had a love-hate relationship with her until I discovered what she could do for me. Siri is super capable, speaks over 20 languages and is always there for you with a simple "Hey Siri".__
+
+Siri got released 4 years ago on October 4, 2011 and is available for iOS, watchOS, tvOS and since Sierra also for macOS. It got physical when the fourth generation Apple TV got introduced with a [Siri Remote](https://en.m.wikipedia.org/wiki/Siri_Remote). And is the only personal assistant that speaks twenty languages, including Chinese, Danish, Japanese, Korean and Malay. Many of us use it daily for ordinary tasks, but I would love to show you my five favorite commands I didn't know existed.
+
+## 1. Currency conversion
+
+You know the drill, you are in a country where you aren't familiar with the local currency and you need to convert it into your own. Or you are shopping for new things online and want to know if you save or lose money ordering it abroad. Siri, with the help of Yahoo, can easily help you with that.
+
+__Hey Siri, what's $200 in euros__
+
+![](/img/2016-10-04-Siri-Currency.JPG)
+
+The current conversion rate gets pulled from the Internet and Siri converts everything for you. It is definitely faster and easier than googling for the conversion rate, especially when you want to focus on your current task.
+
+## 2. Date calculation
+
+Are your kids begging you to think of their Christmas gifts already? Maybe you are on a deadline and want to know how many days you have left to calm your nerves?
+Siri can easily be of good value here. You can ask her to tell you the days until a certain event, the weekday of a date and ask other complex queries that would else take you quite some time to figure out.
+
+Sample Queries:
+
+__Hey Siri, what day was 72 days ago?__
+
+__Hey Siri, how many days until Christmas?__
+
+![](/img/2016-10-04-Siri-Dates.JPG)
+
+## 3. Time
+
+Of course you can look at the clock to find out the time, but sometimes you have to set a meeting in a different country on a different continent in a different time zone and we all know how this might end. To help you, Siri can do the same queries as with Dates. Ask her what the time at home is or how late it is in Singapore. Simple and safe.
+
+__Hey Siri, what time is it in Singapore?__
+
+![](/img/2016-10-04-Siri-Time.PNG)
+
+## 4. Messaging
+
+New in iOS 10 are Messenger extensions for Siri. Many people love dictating messages on their phone. Why not take it to the next level and compose the whole message and send it with your favorite messaging app? This lets you keep your hands on the wheel or in the dough and still text with your friends or coworkers using [WhatsApp](https://itunes.apple.com/en/app/whatsapp-messenger/id310633997?mt=8),  [Telegram](https://itunes.apple.com/en/app/telegram-messenger/id686449807?mt=8) or iMessage.
```

Comment:
Swap these two sentences:
New in iOS 10 are Messenger extensions for Siri. 
Many people love dictating messages on their phone.